nexa/gguf/streamlit/streamlit_image_chat.py

```diff
@@ -1,29 +1,129 @@
 import os
 import sys
+import subprocess
+import re
+from typing import List, Optional
 from PIL import Image
-from nexa.general import pull_model
 import streamlit as st
+from nexa.general import pull_model
 from nexa.gguf.nexa_inference_image import NexaImageInference
 import io
 
-default_model = sys.argv[1]
-is_local_path = False if sys.argv[2] == "False" else True
-hf = False if sys.argv[3] == "False" else True
+# init:
+DEFAULT_PARAMS = {
+    "num_inference_steps": 50,
```

Comment:
50 steps is too slow, and lcm-dreamshaper only takes 4 steps, sd1.5 takes 20 steps
https://github.com/NexaAI/nexa-sdk/blob/main/nexa/gguf/nexa_inference_image.py#L176

also, image generation parameters are predefined here
https://github.com/NexaAI/nexa-sdk/blob/main/nexa/constants.py#L286-L320